lib/validates_zipcode/cldr_regex_collection.rb

```diff
@@ -132,117 +134,120 @@ module CldrRegexpCollection
-      BI: /\A([a-zA-Z\d\s]){3,}\z/,
-      BS: /\A([a-zA-Z\d\s]){3,}\z/,
-      BZ: /\A([a-zA-Z\d\s]){3,}\z/,
+      AG: /\A([A-Z\d\s]){3,}\z/i,
+      AO: /\A([A-Z\d\s]){3,}\z/i,
+      AZ: /\A([A-Z\d\s]){3,}\z/i,
+      BF: /\A([A-Z\d\s]){3,}\z/i,
+      BI: /\A([A-Z\d\s]){3,}\z/i,
+      BS: /\A([A-Z\d\s]){3,}\z/i,
+      BZ: /\A([A-Z\d\s]){3,}\z/i,
       BR: /\A\d{5}(-?\d{3})?\z/,
-      BJ: /\A([a-zA-Z\d\s]){3,}\z/,
+      BJ: /\A([A-Z\d\s]){3,}\z/i,
       BT: /\A\d{5}\z/,
-      BQ: /\A([a-zA-Z\d\s]){3,}\z/,
+      BQ: /\A([A-Z\d\s]){3,}\z/i,
       BO: /\A\d{4}\z/,
-      BW: /\A([a-zA-Z\d\s]){3,}\z/,
-      CF: /\A([a-zA-Z\d\s]){3,}\z/,
-      CG: /\A([a-zA-Z\d\s]){3,}\z/,
-      CI: /\A([a-zA-Z\d\s]){3,}\z/,
-      CM: /\A([a-zA-Z\d]){3,7}\z/,
-      CO: /\A([a-zA-Z\d\s]){3,}\z/,
+      BW: /\A([A-Z\d\s]){3,}\z/i,
+      CF: /\A([A-Z\d\s]){3,}\z/i,
+      CG: /\A([A-Z\d\s]){3,}\z/i,
+      CI: /\A([A-Z\d\s]){3,}\z/i,
+      # NOTE: Cameroon has no postal codes
+      # CM: /\A([A-Z\d]){3,7}\z/i,
+      CO: /\A([A-Z\d\s]){3,}\z/i,
       CR: /\A\d{5}\z/,
       CU: /\A\d{5}\z/,
-      KM: /\A([a-zA-Z\d\s]){3,}\z/,
-      DJ: /\A([a-zA-Z\d\s]){3,}\z/,
-      DM: /\A([a-zA-Z\d\s]){3,}\z/,
-      EG: /\A([a-zA-Z\d\s]){3,}\z/,
-      GQ: /\A([a-zA-Z\d\s]){3,}\z/,
+      KM: /\A([A-Z\d\s]){3,}\z/i,
+      DJ: /\A([A-Z\d\s]){3,}\z/i,
+      DM: /\A([A-Z\d\s]){3,}\z/i,
+      EG: /\A([A-Z\d\s]){3,}\z/i,
+      GQ: /\A([A-Z\d\s]){3,}\z/i,
       EL: /\A\d{5}\z/,
-      ER: /\A([a-zA-Z\d\s]){3,}\z/,
-      FJ: /\A([a-zA-Z\d\s]){3,}\z/,
+      ER: /\A([A-Z\d\s]){3,}\z/i,
+      FJ: /\A([A-Z\d\s]){3,}\z/i,
       FR: /\A\d{5}\z/,
-      GA: /\A\d{2}\z/,
-      GD: /\A([a-zA-Z\d\s]){3,}\z/,
-      GH: /\A([a-zA-Z\d\s]){3,}\z/,
-      GM: /\A([a-zA-Z\d\s]){3,}\z/,
-      GN: /\A([a-zA-Z\d\s]){3,}\z/,
-      GI: /\A(GX11 1AA)\z/,
+      # https://www.gpost.ge/?site-lang=en&site-path=help%2Fzipcodes%2F&group=3
+      GA: /\A\d{4}\z/,
+      GD: /\A([A-Z\d\s]){3,}\z/i,
+      GH: /\A([A-Z\d\s]){3,}\z/i,
+      GM: /\A([A-Z\d\s]){3,}\z/i,
+      GN: /\A([A-Z\d\s]){3,}\z/i,
+      GI: /\A(GX11[ ]?1AA)\z/i,
       GR: /\A\d{3}[ ]?\d{2}\z/,
-      GY: /\A([a-zA-Z\d\s]){3,}\z/,
-      HN: /\A(([a-zA-Z]){2}|\d{2})\d{4}\z/,
+      GY: /\A([A-Z\d\s]){3,}\z/i,
+      HN: /\A(([A-Z]){2}|\d{2})\d{4}\z/i,
       IO: /\A\d{5}\z/,
       IR: /\A\d{5}\z/,
       JP: /\A\d{3}\-?\d{4}\z/,
-      JM: /\A(JM)[A-Z]{3}\d{2}\z/,
-      KI: /\A([a-zA-Z\d\s]){3,}\z/,
-      KN: /\A([a-zA-Z\d\s]){3,8}\z/,
-      KP: /\A([a-zA-Z\d\s]){3,8}\z/,
+      # https://en.youbianku.com/Jamaica#:~:text=Jamaica%20has%20no%20postcode%20system,the%20name%20of%20the%20town.
+      JM: /\A((JM)(DCN|CHR|AKN|DMR|BPD|AAW|CAN|ACE|DEH|CJS|BMY|BTS|CTY|DWD)(0[1-9]|(1|2)[0-9]|3[1-9]))\z/i,
+      KI: /\A(KI)\d{4}\z/i,
+      KN: /\A(KN)\d{4}\z/i,
+      KP: /\A([A-Z\d\s]){3,8}\z/i,
       LI: /\A\d{4}\z/,
-      LV: /\A(LV)?\-?\d{4}\z/,
+      LV: /\A(LV)?\-?\d{4}\z/i,
```

Comment:
may not be required if we want to force `LV` to be uppercase.
 Could also consider allowing `LV 1234` and `LV1234` like with `LU`